src/Misc/layoutbin/update.sh.template

```diff
@@ -118,40 +118,47 @@ then
     exit 1
 fi
 
-# fix upgrade issue with macOS
+# fix upgrade issue with macOS when running as a service
 currentplatform=$(uname | awk '{print tolower($0)}')
-if [[ "$currentplatform" == 'darwin' ]]; then
-    # need a short-term fix for https://github.com/actions/runner/issues/743
-    # we will recreate all the ./externals/node12/bin/node of the past 5 versions
-    # v2.280.3 v2.280.2 v2.280.1 v2.279.0 v2.278.0
-    if [[ ! -e "$rootfolder/externals.2.280.3/node12/bin/node" ]]
-    then
-        mkdir -p "$rootfolder/externals.2.280.3/node12/bin"
-        cp "$rootfolder/externals/node12/bin/node" "$rootfolder/externals.2.280.3/node12/bin/node"
-    fi
-
-    if [[ ! -e "$rootfolder/externals.2.280.2/node12/bin/node" ]]
+if [[ "$currentplatform" == 'darwin'  && restartinteractiverunner -eq 0 ]]; then
```

Comment:
should we rely on `restartinteractiverunner` or `-f $rootfolder/.service`?

Reply:
spoke in person, lets use restartinteractiverunner for now